src/main/java/org/folio/inventory/consortium/util/InstanceOperationsHelper.java

```diff
@@ -0,0 +1,95 @@
+package org.folio.inventory.consortium.util;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import org.apache.http.HttpStatus;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.inventory.consortium.handlers.TenantProvider;
+import org.folio.inventory.dataimport.exceptions.OptimisticLockingException;
+import org.folio.inventory.domain.instances.Instance;
+import org.folio.inventory.exceptions.NotFoundException;
+import org.folio.kafka.exception.DuplicateEventException;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang.StringUtils.isNotBlank;
+import static org.folio.inventory.dataimport.util.DataImportConstants.UNIQUE_ID_ERROR_MESSAGE;
+
+public class InstanceOperationsHelper {
```

Comment:
nit: This class is in the consortium package but i think it can be useful and placed outside of it. I don' t think anythin gin this class is  ECS specific.

Reply:
Do you mean I have to move this class to another package?

Reply:
Yes. You don't have to. It is a suggestion.